server/endpoints/Profile.ts

```diff
@@ -9,10 +9,14 @@ export interface NetIdQuery {
   netId: string;
 }
 
+// The type of a query with a token
 export interface ProfileRequest {
   token: string;
 }
 
+/**
+ * [getStudentEmailByToken] returns a student email if the given token is valid
```

Comment:
This is my bad for not specifying the requirements of a doc string. But, ideally we would have something like this:
```
/**
   * Returns true if [insert something here] matching the email if the given token is valid and false otherwise.
   * This method authenticates the user token through the Google API.
   * @param token: google auth token
   * @requires that you have a handleVerifyError, like as follows:
   * verify(token, function(){//do whatever}).catch(function(error){
   */
```

Something similar to the code I wrote above to specify parameters, return types, etc. Granted this file doesn't do that well or consistently hence the new documentation requirement